pkg/manager/manager_tasks.go

```diff
@@ -626,6 +639,54 @@ func (s *singleton) Clean() {
 	}()
 }
 
+func (s *singleton) MigrateHash() {
+	if s.Status.Status != Idle {
+		return
+	}
+	s.Status.SetStatus(Clean)
+	s.Status.indefiniteProgress()
+
+	qb := models.NewSceneQueryBuilder()
+
+	go func() {
+		defer s.returnToIdleState()
+
+		logger.Infof("Migrating generated files for current naming hash")
+
+		scenes, err := qb.All()
+		if err != nil {
+			logger.Errorf("failed to fetch list of scenes for cleaning")
```

Comment:
even if the task is using the clean state isn't it more helpful to print something like "migrating" or "file renaming" or something like that?